pyproject.toml

```diff
@@ -44,7 +44,9 @@ bandit = "*"
 pylint = "*"
```

Comment:
Can we remove the pinning to a specific `typed-ast` version above? Or is this still an issue with black?

Reply:
Fixing.